test/parallel/xyk-pallet.API.buyAsset.test.ts

```diff
@@ -368,17 +298,16 @@ describe("xyk-pallet - Buy assets tests: Buying assets you can", () => {
     keyring.addPair(testUser2.keyRingPair);
     [firstCurrency, secondCurrency] = await Assets.setupUserWithCurrencies(
       testUser1,
-      [defaultCurrecyValue, defaultCurrecyValue.add(new BN(1))],
+      [defaultCurrencyValue, defaultCurrencyValue.add(new BN(1))],
       sudo
     );
     const [thirdCurrency] = await Assets.setupUserWithCurrencies(
       testUser2,
-      [defaultCurrecyValue],
+      [defaultCurrencyValue],
       sudo
     );
 
     await sudo.mint(thirdCurrency, testUser1, thirdAssetAmount);
-    await testUser1.addMGATokens(sudo);
     await createPool(
       testUser1.keyRingPair,
       firstCurrency,
```

Comment:
On line 316, secondAssetAmount is not the right one.  
Before it was this,
secondAssetAmount.div(new BN(2))
and now is the same, but with half of the value

Reply:
Fixed